app/src/org/commcare/utils/FirebaseMessagingUtil.java

```diff
@@ -46,4 +51,19 @@ public static String removeServerUrlFromUserDomain(String userDomain) {
         }
         return userDomain;
     }
+
+    public static void verifyToken() {
+        if(!BuildConfig.DEBUG) {
```

Comment:
Why not init token in debug ?

Reply:
Nothing in particular, just trying to avoid noise but I'm happy to revert that.

Reply:
yeah we should revert to be able to have receive push notifications with debug apk as well.